### DISCUSSION TITLE
请问如何修改frontmatter中预设的部分？

### DISCUSSION THREAD

**Discussion by Adashaw77** [zhaohongxuan/obsidian-weread-plugin#306] [Q&A]
开始以为在模板中可以直接修改头部yaml属性，但发现似乎这部分是写死的，即使我把模板文件全部删掉置空，导出的笔记还是有预设的头部yaml文件。请问如何编辑这部分呢？
<img width="804" alt="截屏2024-05-16 12 56 58" src="https://github.com/zhaohongxuan/obsidian-weread-plugin/assets/41321836/2b932ba5-58c9-485b-a1b1-db75a26f0f9e">
<img width="804" alt="截屏2024-05-16 12 56 39" src="https://github.com/zhaohongxuan/obsidian-weread-plugin/assets/41321836/4cc87b03-a3a8-42c4-8d07-d276675a61db">

**Comment by 6Lexus9** [comment]
同问
